libs/ledgerjs/packages/cryptoassets/src/currencies.ts

```diff
@@ -479,6 +482,7 @@ export const cryptocurrenciesById: Record<string, CryptoCurrency> = {
       },
     ],
     keywords: ["bsc", "bnb", "binance", "binance smart chain", "binance chain"],
+    explorerId: "bnb",
```

Comment:
Maybe for future integration we could discuss with backend team so that our `id` is the same as the `explorerId`, we could've use the same typing

Reply:
Yes it has been discussed already i need to ping valentin

Reply:
more the other way around btw (backend api to expose an interface with ledger live ids ;) ) since we will never be able to change our ids (immutable because stored on users app.json files)

infra actually had a proposal for this a few years ago, they can implement an aliasing supporting our ids.

Reply:
I discussed it with Valentin and Hakim and it seems like we should use the explorerId for now... maybe for v5